<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Tweets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Documents/atoms30/trunk/PROJECTS/SMGA/MainHandler/assets/graphics/background/floor_line_blue_light.ico</normaloff>Documents/atoms30/trunk/PROJECTS/SMGA/MainHandler/assets/graphics/background/floor_line_blue_light.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>761</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40,150);</string>
   </property>
   <widget class="QLabel" name="label_words">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,0);</string>
    </property>
    <property name="text">
     <string>Words:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_language">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,0);</string>
    </property>
    <property name="text">
     <string>Language</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hastags">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,0);</string>
    </property>
    <property name="text">
     <string>Hastags:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_users">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,0);</string>
    </property>
    <property name="text">
     <string>Users</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,0);</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_words">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>example: home rent loft</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_hastags">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>example: #euros #PedroSanchez</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_language">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>example: ES</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_users">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>641</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>example: @angeldambrosiof @A3Noticias</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_until">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2021-10-10</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_since">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>2020-10-10</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-240</x>
     <y>30</y>
     <width>1901</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>.designer/graphics/necktie-angle-pattern-vector-cartoon-pretty-wavy-lines-b5f550c08c0bdaf5050b2af8e6b6f0a8.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Documents/atoms30/trunk/PROJECTS/SMGA/MainHandler/assets/graphics/background/floor_white.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40,0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; color:#000000;&quot;&gt;Twitter Advanced Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>670</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40,0);</string>
   </property>
   <property name="text">
    <string>Loading:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loading">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>670</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(78, 154, 6);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt;</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_manual_query">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40,0);</string>
   </property>
   <property name="text">
    <string>Manual Query</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_manual_query">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>751</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="start_manual_query">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt;</string>
   </property>
   <property name="text">
    <string>Start Manual Query</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_limit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>460</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40,0);</string>
   </property>
   <property name="text">
    <string>Limit:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_limit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>5000</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_8</zorder>
  <zorder>widget_1</zorder>
  <zorder>label_4</zorder>
  <zorder>label_17</zorder>
  <zorder>start_button</zorder>
  <zorder>label_loading</zorder>
  <zorder>stop_button</zorder>
  <zorder>label_manual_query</zorder>
  <zorder>line_manual_query</zorder>
  <zorder>start_manual_query</zorder>
  <zorder>label_limit</zorder>
  <zorder>line_limit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
